public/Stations.php

```diff
@@ -52,12 +51,16 @@ function get($id) {
    * station model itself!
    *
    * @param int $id The station ID to calculate related onramp ID for
+   * @return stdClass
```

Comment:
I like adding this return! I think adding a little description of what the stdClass returned is expected to have would be an excellent addition as well.
```php
@return stdClass $retVal An object containing the ID searched for, the calculated result, and an array of relation onramps.
```